php/libraries/Candidate.class.inc

```diff
@@ -9,6 +9,7 @@
 use LORIS\StudyEntities\Candidate\CandID;
 use LORIS\StudyEntities\Candidate\EntityType;
 use LORIS\StudyEntities\Candidate\Sex;
+use LORIS\StudyEntities\Candidate\ParentIDGenerator;
```

Comment:
not sure if the ParentIDGenerator should be classed within StudyEntities. I think it makes sense for ParentID to be a candidate study entity, but not the generator itself

Reply:
i think you can create a LORIS\StudyEntities\Candidate\ParentID class that looks exactly like LORIS\StudyEntities\Candidate\CandID, extending the ValidatableIdentifier abstract class, and specififying `return 'ParentID'` within the getType() method